### ISSUE TITLE
Missing GET parameters in query

### ISSUE THREAD

**Issue by restanrm** [gbossert/APITurntable#16]
`uri_append` doesn't contains the GET parameters of the query: 

Ex: 
**request:** `curl http://localhost:4242/pivot/lieghih2thaiMuz/play/all\?tag`
**received:** 
```bash
{
  "method": "GET",
  "uri_append": "play/all",
  "headers": {
    "Host": "localhost",
    "X-Real-Ip": "172.26.0.1",
    "X-Forwarded-For": "172.26.0.1",
    "X-Forwarded-Proto": "http",
    "Connection": "close",
    "User-Agent": "curl/7.59.0",
    "Accept": "*/*"
  },
  "body": "",
  "cookies": {},
  "source_ip": "172.26.0.5",
  "source_url": "http://localhost/pivot/lieghih2thaiMuz/play/all?tag"
}
```

The `tag` parameter is only accessible through the source URL, and not easily extractible. It would be better if the `uri_append` contains the `GET` parameter or if we could have the basepath of the URL of the pivot.

**Comment by otetard** [comment]
This issue should be fixed with 3c3c27c90344c5e9e7975b19e6a4597ff1c0ba33.

**Comment by otetard** [comment]
Close that issue since 3c3c27c90344c5e9e7975b19e6a4597ff1c0ba33 was merged.
